website_snippet_timeline/templates/snippets.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2018 Onestein
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+    <template id="snippets" inherit_id="website.snippets">
+        <xpath expr="//div[@id='snippet_feature']/div[hasclass('o_panel_body')]" position="inside">
+            <t t-snippet="website_snippet_timeline.snippet_timeline"
+               t-thumbnail="/website_snippet_timeline/static/src/img/snippet.png" />
+
+            <t t-snippet="website_snippet_timeline.snippet_timeline_item"
+               t-thumbnail="/website_snippet_timeline/static/src/img/snippet-item.png" />
+        </xpath>
+    </template>
+
+    <template id="snippet_options" inherit_id="website.snippet_options">
+        <xpath expr="." position="inside">
+            <div data-selector=".snippet_timeline_item"
+                 data-js="timeline_item"
+                 data-drop-in=".snippet_timeline ul" data-drop-near=".snippet_timeline ul > li" ></div>
+
+            <div data-js="margin-y"
+                 data-selector=".snippet_timeline_item">
+            </div>
+        </xpath>
+    </template>
+
+    <template id="snippet_timeline_item" name="Timeline - Item">
```

Comment:
prefix module name or "Snippet" to templates' names to ease lookup in the backend

Reply:
The problem when I change the name here that the label in the block menu is also changed:

![image](https://user-images.githubusercontent.com/10028499/48539929-eea1ae80-e8b8-11e8-9e12-abfae8317b9a.png)